azure-pipelines-product-construction-service.yml

```diff
@@ -85,6 +87,19 @@ stages:
               Set-Content -Path $(diffFolder)/before.json -Value $before
           displayName: Snapshot configuration (before)
 
+        - task: AzureCLI@2
+          name: GetAuthInfo
+          displayName: Get auth information
+          inputs:
+            azureSubscription: $(authServiceConnection)
+            addSpnToEnvironment: true
+            scriptType: pscore
+            scriptLocation: inlineScript
+            inlineScript: |
+              # Fetch pcs token
```

Comment:
This is technically Maestro's token, no?

Reply:
I mean they're the same, we're using it for PCS in this case. Think it'd be confusing if it said Maestro

Reply:
It says Maestro in the service connection, but I get where you are coming from